desktop/cmp/record/RecordActionBar.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {PropTypes as PT} from 'prop-types';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {hbox} from '@xh/hoist/cmp/layout';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {RecordAction} from '@xh/hoist/cmp/record';
+
+import './RecordActionBar.scss';
+
+@HoistComponent
+export class RecordActionBar extends Component {
+    baseClassName = 'xh-record-action-bar';
+
+    static propTypes = {
+        /** RecordAction or configs to create. */
+        actions: PT.arrayOf(PT.oneOfType([PT.object, PT.instanceOf(RecordAction)])).isRequired
+    };
+
+    actions = [];
+
+    constructor(props) {
+        super(props);
+
+        this.actions = props.actions.filter(it => it !== null).map(it => new RecordAction(it));
```

Comment:
This always creates a RecordAction, but prop type claims that application can pass in one 'already built'.